### PR TITLE
Limit headings to h2, h3, h4

### PR DESCRIPTION
**Pull request by cindywu**
Removes `h1`, `h5`, `h6` from the editor (removes keyboard shortcuts, markdown).

fixes one item in https://github.com/jellypbc/poster/issues/232

image

We can bring back `h5`, `h6` in the future if needed. To do so those headings require font size styles.

### CODE REVIEW

File: app/javascript/components/editor-config/keys.js [hunk 1]
```diff
@@ -62,12 +62,9 @@ const keys = {
   'Mod-]': sinkListItem(schema.nodes.list_item),
   'Shift-Ctrl-0': setBlockType(schema.nodes.paragraph),
   'Shift-Ctrl-\\': setBlockType(schema.nodes.code_block),
```
Comment (dennydennydenny) [inline app/javascript/components/editor-config/keys.js:64]: This removes that key for both editors? Is that intended? I think we should be preserving `shift-ctrl-1`.

**Review by dennydennydenny — COMMENTED** [review]
See comment for my question, but restated: are we trying to treat the page with just one `h1`, which is the title editor? Or what if pages end up being parsed from grobid with `h1`, and we shift all headings down one?

File: app/javascript/components/editor-config/keys.js (hunk 1 above)
Comment (cindywu) [reply app/javascript/components/editor-config/keys.js:64]: > This removes that key for both editors? Is that intended?

My intention is to remove `h1` keys from both the title editor and body editor.

Note in a separate PR https://github.com/jellypbc/poster/pull/315 the title editor is given a separate set of keys. The keys in this file are specific to the body editor. 

> are we trying to treat the page with just one h1, which is the title editor?

This is my intention, yes.

> Or what if pages end up being parsed from grobid with h1, and we shift all headings down one?

When we parse PDFs with grobid, I would like to see the biggest heading be `h2`. 

@dennydennydenny would you still like me to put `h1` keys back for the body editor?

Comment (cindywu) [reply app/javascript/components/editor-config/keys.js:64]: the title editor is given a limited set of keys in https://github.com/jellypbc/poster/pull/316

```
const titleKeys = {
  'Mod-z': undo,
  'Shift-Mod-z': redo,
  'Mod-y': redo,
  'Mod-i': toggleMark(schema.marks.em),
  'Mod-Shift-m': addAnnotation,
}
```

Comment (dennyluan) [reply app/javascript/components/editor-config/keys.js:64]: okidoke :+1:

**Pull request merged**